### DISCUSSION TITLE
参考文献采用著者年份制时可以在后边加上编号吗？

### DISCUSSION THREAD

**Discussion by sastars9** [nju-lug/NJUThesis#182] [Q&A] [answered]
想用著者年份制的引用，按姓名排序，但是又想按姓名排序有编号[1]这类的，能做到吗？谢谢

**Comment by dreistein543** [comment, marked as answer]
<img width="835" alt="image" src="https://user-images.githubusercontent.com/30745407/225628872-30e2dd18-a17b-4607-ba25-5409ce21f64c.png">
参考红圈中的设置

**Reply by sastars9** [reply]
![image](https://user-images.githubusercontent.com/128053942/225835701-bd716ee6-1d01-40c6-80c2-e9c251eb21cf.png)
谢谢，设置了可以，这个编号空格较大，还有办法收紧一点吗？

**Reply by dreistein543** [reply]
由于学位论文参考文献数量很多，这是为了与2、3位数对齐拓宽的

**Reply by sastars9** [reply]
感谢，已解决

**Reply by atxy-blip** [reply]
我更倾向于 `gbalign=right`。

**Reply by zijuzhang** [reply]
> 感谢，已解决

这个在哪里设置呀

**Reply by sastars9** [reply]
setup文件里啊
